packages/headless/src/controllers/core/facets/facet/headless-core-facet.ts

```diff
@@ -403,6 +415,9 @@ export function buildCoreFacet(
 
     get state() {
       const request = getRequest();
+      if (!request) {
+        return deletedFacetState;
+      }
```

Comment:
small patch to support when facet get deleted

Reply:
Do you have another example of 'static' state being used somewhere in Headless?

I'm wondering if it should be in the function or module scope 🤔

Reply:
Instead of this, I decided to put the logic for this inside of the `facetRequestSelector` in the `facet-set-selector.ts` instead. Thank you for the comment.